src/main/scala/net/snowflake/spark/snowflake/Parameters.scala

```diff
@@ -131,6 +131,7 @@ object Parameters {
 
   // Proxy related info
   val PARAM_USE_PROXY: String = knownParam("use_proxy")
+  val PARAM_PROXY_PROTOCOL: String = knownParam("proxy_protocol")
```

Comment:
if it only works with AWS, should we name it `aws_proxy_protocol`?
Or do we have a plan to support azure and gcp later?

Reply:
This name is from JDBC.
Currently, it only support AWS. It may support Azure in the future. So it's better to use the same name.